### PR TITLE
Managedccfs command group

### PR DESCRIPTION
**Pull request by msftsettiy**
This PR contains the implementation of the managedccfs command group and the related test cases.  It adds a new command group under the confidentialledger extension. The following commands are implemented:

- create
- delete
- show
- list

The update command is not supported in the public preview phase. Test cases are added to create, view and delete instances using different command arguments.

### CODE REVIEW

**Comment by yonzhan** [comment]
Managedccfs

**Comment by kairu-ms** [comment]
Hi @msftsettiy, could you fix the CI issues?

File: linter_exclusions.yml [hunk 1]
```diff
@@ -319,6 +319,8 @@ codespace plan create:
     default_sku_name:
       rule_exclusions:
       - option_length_too_long
+confidentialledger managedccfs update:
+- missing_command_test_coverage
```
Comment (necusjz) [inline linter_exclusions.yml:323]:
```suggestion
confidentialledger managedccfs update:
  rule_exclusions:
  - missing_command_test_coverage
```
After I reconsidered, it should be modified like that.

File: src/managedccfs/azext_managedccfs/aaz/latest/confidentialledger/managedccfs/_update.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+            options=["--member-identity-certificates"],
+            arg_group="Properties",
+            help="List of member identity certificates for  Managed CCF",
+        )
+        _args_schema.node_count = AAZIntArg(
+            options=["--node-count"],
+            arg_group="Properties",
+            help={"short-summary": "Number of CCF nodes in the instance.", "long-summary": "If the argument is omitted, a default value of 3 is used. The maximum supported size is 9 nodes."},
+            is_preview=True,
+            default=3,
```
Comment (necusjz) [inline src/managedccfs/azext_managedccfs/aaz/latest/confidentialledger/managedccfs/_update.py:89]: Update commands should not have parameters with default values. Let's change it in the workspace and re-generate.

File: src/managedccfs/azext_managedccfs/custom.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            required=True,
+        )
+
+        _Element.group = AAZStrArg(
+            options=["group"],
+            help="A group name for the member.",
+            required=False,
+        )
+
+        args_schema.member_identity_certificates._registered = False
```
Comment (necusjz) [inline src/managedccfs/azext_managedccfs/custom.py:53]: Refer to Pylint W0212 to suppress it.

**Comment by necusjz** [comment]
Please update https://github.com/msftsettiy/azure-cli-extensions-fork/blob/457722ee37c681fb1a49058b3788c0364b784ec6/.github/CODEOWNERS as well.

File: linter_exclusions.yml (hunk 1 above)
Comment (msftsettiy) [reply linter_exclusions.yml:323]: Fixed

**Review by necusjz — APPROVED** [review]

**Pull request merged**